docs/source/storage/general_storage.rst

```diff
@@ -16,20 +24,15 @@ You can schedule an online consultation on the `RC website <https://rc.northeast
    If you have an account on Discovery, see :ref:`discovery_storage`
    for details on the storage available to you specifically for use with Discovery's compute resources. The options listed below are not connected to Discovery.
 
-**NAME:** ``/research``
-  - **DESCRIPTION:** This storage tier is intended to be a repository for data derived from equipment such as lab machines,
-    instruments, etc. The performance capabilities are not intended for parallel or high performance workloads.
-    Data are backed up and a second copy is created. You can request storage on ``/research`` by submitting a `New Storage Space request <https://bit.ly/NURC-NewStorage>`_.
-    See the section "Connecting to /research" below for information on how to connect to this storage.
-  - **QUOTA:** Each group can request up to **35TB** of free storage across all supplemental storage tiers: ``/work/<groupname>``, ``/research`` () and ``/nese``.  
 
+DEPRECATED |ss| **NAME:** ``/research`` |se|
+  - **DESCRIPTION:** This storage tier is no longer provided. Please contact Research Computing if you are a former user of /research and have questions or issues related to this by submitting a ticket https://bit.ly/NURC-Assistance. Other storage options include /work (more information found here :ref:`discovery_storage` and requested with https://bit.ly/NURC-NewStorage), Sharepoint - https://service.northeastern.edu/tech?id=kb_article&sysparm_article=KB0012695, and OneDrive - https://service.northeastern.edu/tech?id=kb_article&sysparm_article=KB0012764. 
```

Comment:
No need to mention "by submitting a new storage request" since we link to the other page. Can we also turned all those links into inline (vs explicit) urls?